src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -33,9 +33,10 @@
             <Setter Property="Template">
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
-                        <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                        <StackPanel Margin="8,0,8,8">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding Path=ActualWidth, 
+                                                      RelativeSource={RelativeSource AncestorType=StackPanel}}" />
```

Comment:
Instead of using type `StackPanel`, can we use an element name instead? That way it will be less error-prone, should another new `StackPanel` is introduced along the visual tree in the future for some reasons. Also the intention of *these two elements are related in width* will be clearer if we have a named element, and this code that depends on that named element.